misc/python/materialize/checks/all_checks/source_tables.py

```diff
@@ -0,0 +1,257 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+from textwrap import dedent
+
+from materialize.checks.actions import Testdrive
+from materialize.checks.checks import Check
+from materialize.checks.executors import Executor
+from materialize.mz_version import MzVersion
+from materialize.mzcompose.services.mysql import MySql
+
+
+class TableFromSourceBase(Check):
+
+    def _can_run(self, e: Executor) -> bool:
+        return self.base_version >= MzVersion.parse_mz("v0.116.0-dev")
+
+    def generic_setup(self) -> str:
+        return dedent(
+            """
+                $ postgres-execute connection=postgres://mz_system:materialize@${testdrive.materialize-internal-sql-addr}
+                ALTER SYSTEM SET enable_create_table_from_source = true
```

Comment:
Can we enable this by default in all of CI?

Reply:
This is being done in the other PR.